fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -172,6 +173,11 @@ default void refreshMaterializedView(String dbName, String mvName, int priority)
             throws DdlException, MetaNotFoundException {
     }
 
+    default void refreshMaterializedView(RefreshMaterializedViewStatement refreshMaterializedViewStatement,
```

Comment:
if you add this function, is the one void refreshMaterializedView(String dbName, String mvName, int priority) useful?

Reply:
RefreshMaterializedView may used by other inner function，it is not suitable to wrap parameters to statement in inner function，so I remain refreshMaterializedView(String dbName, String mvName, int priority) method ，to refresh all partitions and not force。